### PR TITLE
Apple page clone

### PR DESCRIPTION
**Pull request by RICKCOYL**
This is a Apple website article page clone using HTML and CSS3.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          <a href=""><i class="fab fa-apple"></i></a>
+        </li>
+        <li class="nav-items"><a href="">Store</a></li>
+        <li class="nav-items"><a href="">Mac</a></li>
+        <li class="nav-items"><a href="">IPod</a></li>
+        <li class="nav-items"><a href=""> IPhone</a></li>
+        <li class="nav-items"><a href="">IPad</a></li>
+        <li class="nav-items"><a href="">ITunes</a></li>
+        <li class="nav-items"><a href="">Support</a></li>
+        <li class="nav-items"><input type="text" /></li>
```
Comment (peterrobert) [inline index.html:28]: Great work with making these links. However, If you check the original webpage there is a hover effect once you hover the elements kindly add that

File: index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      <div class="second-bg-images">
+        <img src="img/promo_iphone5c.jpg" alt="iphone5c" />
+      </div>
+      <div class="second-bg-images">
+        <img src="img/promo_30_years.jpg" alt="promo" />
+      </div>
+      <div class="second-bg-images">
+        <img src="img/promo_supplier_responsibility.jpg" alt="promo suplier" />
+      </div>
+    </div>
```
Comment (peterrobert) [inline index.html:56]: These images should be links. Kindly introduce anchor tags

File: index.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        <li class="nav-items"><a href="">Mac</a></li>
+        <li class="nav-items"><a href="">IPod</a></li>
+        <li class="nav-items"><a href=""> IPhone</a></li>
+        <li class="nav-items"><a href="">IPad</a></li>
+        <li class="nav-items"><a href="">ITunes</a></li>
+        <li class="nav-items"><a href="">Support</a></li>
+        <li class="nav-items"><input type="text" /></li>
+      </ul>
+    </nav>
+    <h1 class="headings"><img src="img/page_title.png" alt="" /></h1>
```
Comment (peterrobert) [inline index.html:31]: This should not be an image it should be text

File: index.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    <h2 class="headings">
+      <img src="img/subtitle.png" alt="" />
+    </h2>
```
Comment (peterrobert) [inline index.html:34]: This should not be an image. It should be text.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+}
+
+.copyright {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.copyright a:hover {
+  text-decoration: underline;
+}
```
Comment (andresporras3423) [inline styles.css:174]: Same style definition of the selector on line 163. Group selectors, or create new class.

**Pull request merged**